assessments-client/pom.xml

```diff
@@ -6,7 +6,7 @@
 
   <groupId>com.transformuk.hee.tis</groupId>
   <artifactId>assessments-client</artifactId>
-  <version>1.1.1</version>
+  <version>1.1.2</version>
```

Comment:
Sure thing, thanks for picking it up

Reply:
Its interesting - there's a version for the project as a whole (1.0.1). Do you think that needs bumping up? I'm assuming not, since the structure of the project as a whole has not changed, and there no reference to the versioning of its submodules. It was last bumped from 1.0.0 to 1.0.1 in Nov 2018!